utils/wget.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<interface uri="http://repo.roscidus.com/utils/wget" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd" xmlns="http://zero-install.sourceforge.net/2004/injector/interface">
+  <name>GNU Wget</name>
+  <summary xml:lang="en">static builds of Wget with taskbar progressbar Windows certificate store support, manual</summary>
```

Comment:
In principle this is a feed for any kind of Wget build. It currently contains only statically linked Windows builds, but that could change. Therefore I think I would be better to replace this summary with the one from the `<entry-point>` ("A command-line utility for...").

Reply:
done